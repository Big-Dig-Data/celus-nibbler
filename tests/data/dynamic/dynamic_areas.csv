
2020,
Metric,Jan,Feb,Mar
M1,1,2,3
M2,4,5,6

2021,
Metric,Jan,Feb,Mar
M1,2,3,4
M2,5,6,7

2022,
Metric,Jan,Feb,Mar
M1,3,4,5
M2,6,7,8
