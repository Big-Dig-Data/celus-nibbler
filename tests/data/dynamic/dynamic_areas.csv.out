"2020-01-01","2020-01-31","","","","","","","","","M1","1"
"2020-02-01","2020-02-29","","","","","","","","","M1","2"
"2020-03-01","2020-03-31","","","","","","","","","M1","3"
"2020-01-01","2020-01-31","","","","","","","","","M2","4"
"2020-02-01","2020-02-29","","","","","","","","","M2","5"
"2020-03-01","2020-03-31","","","","","","","","","M2","6"
"2021-01-01","2021-01-31","","","","","","","","","M1","2"
"2021-02-01","2021-02-28","","","","","","","","","M1","3"
"2021-03-01","2021-03-31","","","","","","","","","M1","4"
"2021-01-01","2021-01-31","","","","","","","","","M2","5"
"2021-02-01","2021-02-28","","","","","","","","","M2","6"
"2021-03-01","2021-03-31","","","","","","","","","M2","7"
"2022-01-01","2022-01-31","","","","","","","","","M1","3"
"2022-02-01","2022-02-28","","","","","","","","","M1","4"
"2022-03-01","2022-03-31","","","","","","","","","M1","5"
"2022-01-01","2022-01-31","","","","","","","","","M2","6"
"2022-02-01","2022-02-28","","","","","","","","","M2","7"
"2022-03-01","2022-03-31","","","","","","","","","M2","8"
